CODE_COMMENT_STYLE.md

```diff
@@ -0,0 +1,116 @@
+# Code Comment Style
+
+This document describes the code comment style applied to TiKV repositories. When you are to commit, be sure to follow the style to write good code comments.
+
+## Why does a good comment matter?
+
+- To speed up the reviewing process
+- To help maintain the code
+- To improve the API document readability
+- To improve the development efficiency of the whole team
+
+## Where/When to comment?
+
+Write a comment where/when there is context that is missing from the code or is hard to deduce from the code. Specifically, use a comment:
+
+- For important code
+- For obscure code
+- For tricky or interesting code
+- For a complex code block
+- If a bug exists in the code but you cannot fix it or you just want to ignore it for the moment
+- If the code is not optimal but you don’t have a smarter way now
+- To remind yourself or others of missing functionality or upcoming requirements not present in the code
+
+A comment is generally required for:
+
+- Module
+- Type
+- Constant
+- Function
+- Method
+- Variable
+- Typical algorithm
+- Exported name
+- Test case
+- TODO
+- FIXME
+
+## How to comment?
+
+### Format of a good comment (for Rust)
+
+- Non-doc comment
+
+    - Used to document implementation details
+    - Use **//** for a line comment
+    - Use **/* … */** for a block comment (used only when needed)

```

Comment:
I don't see why the rules about block comments should be different for doc comments and non-doc comments. Both cases are aesthetic choices for the eyes of maintainers.

I'm sympathetic to block comments, but I think it would be more consistent to generally disallow block comments everywhere. Maybe with a note indicated possible exception cases, like for commenting out code that for some reason still needs to be checked in; or for cases where you _really_ _really_ want to write with block comments for personal reasons; or temporarily for drafting long-form comments, prior to converting them to line comments.

Reply:
Updated as:   
 **Note**: Block comments are not recommended unless for personal reasons or temporary purposes prior to being converted to line comments)